docs/general/logging.md

```diff
@@ -12,3 +12,9 @@ You may customize the logger in an initializer, for example:
 ```ruby
 ActiveModelSerializers.logger = Logger.new(STDOUT)
 ```
+
+You can also disable the logger, just put this in `config/application.rb`:
+
+```ruby
+ActiveSupport::Notifications.unsubscribe(ActiveModelSerializers::Logging::RENDER_EVENT)
```

Comment:
@MSathieu not exactly. It's actually the call to `Bundle.require` at the top of most `config/application.rb` and then only if the gem doesn't have `require: false`.

Also, relying on autoloading of your project's classes is pretty much required for code reloading to work. But that doesn't really apply to gems.  It's a good idea to require a gem before you use it. But this opinion is controversial enough that I won't tell you to include it :)